### ISSUE TITLE
Change asserts to raised errors.

### ISSUE THREAD

**Issue by caternuson** [adafruit/Adafruit_CircuitPython_TCS34725#11]
Change the assert checks to raise an appropriate Error along with helpful message.

**Comment by sommersoft** [comment]
To help explain a little in case this gets picked up during #Hacktoberfest, @caternuson is suggesting to change instances like this:

```python
@integration_time.setter
    def integration_time(self, val):
        assert 2.4 <= val <= 614.4
```
to something like this:
```python
@integration_time.setter
    def integration_time(self, val):
        if 2.4 <= val <= 614.4:
            raise ValueError("Integration Time must be between '2.4' and '614.4'")
```

**Comment by VladMihai28** [comment]
Hi, I'd like to take this issue please.

**Comment by caternuson** [comment]
Fixed by #12. Thanks!
